source/tags/PlaceholderSvg/PlaceholderSvg.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PlaceholderSvg from './PlaceholderSvg';
+
+export default [{
+  name: "default",
+  component: (
+    <PlaceholderSvg width={600} height='300' />
```

Comment:
@krambuhl I left this as is and pushed what I had updated. I'll fix it the quote blizzard here on a later commit once I'm clearer on what needs to happen.